packages/react-utilities/src/compose/resolveShorthand.ts

```diff
@@ -33,7 +33,9 @@ export const resolveShorthand: ResolveShorthandFunction = (value, options) => {
   if (typeof value === 'string' || typeof value === 'number' || Array.isArray(value) || isValidElement(value)) {
     resolvedShorthand.children = value;
   } else if (typeof value === 'object') {
-    resolvedShorthand = value;
+    // TODO: "Type 'readonly ReactNode[]' has no properties in common with type 'UnknownSlotProps'."
```

Comment:
I'm not sure what happened here, but someone more familiar with these types should probably look into it and fix properly